perses/app/relative_point_mutation_setup.py

```diff
@@ -141,6 +144,8 @@ def __init__(self,
                 map of new to old sidechain atom indices to add to the default map (by default, we only map backbone atoms and CBs)
             demap_CBs : bool, default False
                 whether to remove CBs from the mapping
+            solvate : bool, default True
```

Comment:
can you clarify _when_ solvation happens. perses does solvate protein and protein:ligand systems. i think the difference is _when_ you are solvating, or rather, whether you have provided a pre-solvated pdb/structure file. perhaps this should change the name of the argument you are using. can you put into the docs that if `False`, it should be pre-solvated. Also, i was going to suggest adding a sanity check that will check if there are waters in the struct file/topology and raise an error if there are waters, but one has already specified `solvate=True`. Then again, if the pdb contains xtal waters that are important, that error would be erroneous. maybe adding a sanity exception error if there are lots of waters in the pdb structure.

Reply:
>can you put into the docs that if False, it should be pre-solvated.
yes

> Also, i was going to suggest adding a sanity check that will check if there are waters in the struct file/topology and raise an error if there are waters, but one has already specified solvate=True. Then again, if the pdb contains xtal waters that are important, that error would be erroneous.
Yes, I thought about throwing an error like this, but like you said, if there are crystal waters present, but the PDB is not actually solvated, the error would still be present, which we don't want.